syntax = "proto3";

message MxGpioStore {
  repeated bool state = 1;
}
